src/CONST.js

```diff
@@ -910,6 +910,7 @@ const CONST = {
         ES: 'es',
 
         DEFAULT: 'en',
+        ES_ES_ONFIDO: 'es_ES',
```

Comment:
I like its place below the `ES_ES`; it's easy to notice the difference. Let's leave `DEFAULT` alone 😄